opal/datatype/opal_datatype_copy.c

```diff
@@ -57,7 +57,9 @@
 static void *opal_datatype_accelerator_memcpy(void *dest, const void *src, size_t size)
 {
     int res, dev_id;
-    if (NULL != opal_accelerator.memcpy) {
+    uint64_t flags;
+    if (opal_accelerator.check_addr(src, &dev_id, &flags) > 0 || 
```

Comment:
I think I pointed out previously that if we're already doing address detection, we should store this info and pass it along instead of MCA_ACCELERATOR_TRANSFER_UNSPEC

Reply:
done

Reply:
done